rpcd/playbooks/verify-maas.yml

```diff
@@ -23,7 +23,7 @@
         . {{ maas_venv_bin }}/activate
         {% endif %}
         {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
-        --entity {{ inventory_hostname }} \
+        --entity {{ inventory_hostname }}{{ maas_fqdn_extension }} \
```

Comment:
The issue we face is that we cannot change the templates used by setup maas without upgrade impact. Although that inconsistency needs to be cleaned up, it does not cause and failures as far as I am aware. The "--entity" option here needs needs to have the same value as when the entity is setup in the host_setup.yml task, or verify_mass.yml will fail. As you said, the maas_fqdn_extension fact is blank by default, which is why this script does not fail under default builds. However, there were examples of error (as can be seem from Thomas's comment on issue 190) caused by this in environments where the maas_fqdn_extension is set.